### PR TITLE
old man

### PR DESCRIPTION
**Pull request by YuliaPozova**

### CODE REVIEW

File: src/test/java/Main.java [hunk 1]
```diff
@@ -38,7 +38,7 @@ public void test() {
             e.printStackTrace();
         }
     }
-
+//hello
```
Comment (Andrey9417) [inline src/test/java/Main.java:41]: привет

**Review by Andrey9417 — APPROVED** [review]

File: src/test/java/Main.java (hunk 1 above)
Comment (YuliaPozova) [reply src/test/java/Main.java:41]: на пиво нет

File: src/test/java/Main.java [hunk 2]
```diff
@@ -38,7 +38,7 @@ public void test() {
...
     @AfterTest
     public void quitdriver() {
         driver.quit();
```
Comment (Andrey9417) [inline src/test/java/Main.java:44]: why old man?

**Review by Andrey9417 — CHANGES_REQUESTED** [review]

File: src/test/java/Main.java (hunk 2 above)
Comment (YuliaPozova) [reply src/test/java/Main.java:44]: because you old

Comment (YuliaPozova) [reply src/test/java/Main.java:44]: and angry

Comment (Andrey9417) [reply src/test/java/Main.java:44]: image

**Review by Andrey9417 — APPROVED** [review]

File: src/test/java/Main.java (hunk 2 above)
Comment (YuliaPozova) [reply src/test/java/Main.java:44]: cool

**Pull request merged**